packages/wallets/nightly/src/adapter.ts

```diff
@@ -0,0 +1,174 @@
+import {
+    BaseSignerWalletAdapter,
+    scopePollingDetectionStrategy,
+    WalletAccountError,
+    WalletConnectionError,
+    WalletDisconnectedError,
+    WalletError,
+    WalletName,
+    WalletNotConnectedError,
+    WalletNotReadyError,
+    WalletPublicKeyError,
+    WalletReadyState,
+    WalletSignTransactionError,
+} from '@solana/wallet-adapter-base';
+import { PublicKey, Transaction } from '@solana/web3.js';
+
+interface SolanaNightly {
+    publicKey: PublicKey;
```

Comment:
Should this be `publicKey?: PublicKey;` or something? Wondering what the value is when the wallet isn't connected since it's accessed below

Reply:
it's set to '11111111111111111111111111111111' when not connected

Reply:
Isn't it the case then that the check on L99 (`!wallet.publicKey`) will never fail, even if there's no wallet to connect to?

Reply:
I must have overlooked it; now I've changed the condition, I hope everything will be fine now